CONTRIBUTING.md

```diff
@@ -435,6 +436,13 @@ ghstack submit
 [`ghstack`](https://github.com/ezyang/ghstack). It creates a large commit that is
 of very low signal to reviewers.
 
+## Merging your Change
+If you know the right people or team that should approve your PR, add them to the Reviewers list.
+
+If not, your PR will get triaged and assigend to the appropriate person in a couple business days.
+
+Once your PR is approved, you can merge it in by entering a comment with the content `@pytorchmergebot merge`
```

Comment:
Do you think adding a link to the bot wiki here would be helpful?

Reply:
yes :D